app-main/public/js/lessons/markdown_export.js

```diff
@@ -0,0 +1,203 @@
+function ExportToMarkdown() {
+  //Retrieves the lesson object from localstorage given the lessonID
+  this.dataFromLocalStorage = function (lessonID) {
+    try {
+      return JSON.parse(localStorage.getItem("user")).lessons.filter(
+        (lesson) => lesson.id === lessonID
+      )[0];
+    } catch {
+      alert("Something went wrong. The note could not be found!");
+    }
+  };
+
+  //Takes a lesson object and formats the data to be markdown
+  this.formatMarkdown = function (lessonObject) {
```

Comment:
Destructuring can be a nice clean alternative for extracting values from object props. I.e. 

`function ({ title, content }) {`